src/Documentation/sidebar.json

```diff
@@ -96,7 +96,8 @@
       "run.md",
       "status.md",
       "unlock.md",
-      "unprotect.md"
+      "unprotect.md",
+      "cache.md"
```

Comment:
add a label below, keep this list alphabetically sorted

Reply:
Done. It seems like we could easily generate this by simply listing contents of the docs/command-reference directory. Same with labels, which could be parsed from .md header. But that is definitely for the future.